examples/example-webhooks-slack/manage_webhooks.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+"""
+An example script showing ways to manage your webhooks using the
+python-asana client library, see README.md or run ./manage_webhooks.py -h
+for usage examples
+"""
+
+import argparse
+import os
+
+import asana
+
+WORKSPACE_ID = "15793206719"
+
+client = asana.Client.access_token(os.environ.get("ASANA_ACCESS_TOKEN"))
+WORKSPACE_NAME = client.workspaces.find_by_id(WORKSPACE_ID).get("name")
+
+
+def list_webhooks():
+    print(
+        "Displaying webhooks you own associated with workspace: {}".format(
+            WORKSPACE_NAME
+        )
+    )
+    webhooks = client.webhooks.get_all({"workspace": WORKSPACE_ID})
+    for w in webhooks:
+        print(w)
+
+
+def delete_webhook(gid):
+    print("Deleting webhook: {}".format(gid))
+    client.webhooks.delete_by_id(gid)
+
+
+def delete_all_webhooks():
+    print(
+        "Deleting all webhooks you own associated with workspace: {}".format(
+            WORKSPACE_NAME
+        )
+    )
+    webhooks = client.webhooks.get_all({"workspace": WORKSPACE_ID})
+    for w in webhooks:
+        client.webhooks.delete_by_id(w.get("gid"))
+
+
+def create_webhook(resource, target):
+    print(
+        "Creating webhook on {}, make sure that the server at {} is ready to accept requests!".format(
+            resource, target
+        )
+    )
+    client.webhooks.create({"resource": resource, "target": target})
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    subparsers = parser.add_subparsers(
+        help="What you want to do with Asana webhooks", dest="command", required=True
+    )
+
+    list_parser = subparsers.add_parser("list", help="List existing webhooks")
+
+    create_parser = subparsers.add_parser("create", help="Create a new webhook")
+    create_parser.add_argument(
+        "--resource",
+        type=str,
+        help="A resource ID to subscribe to. The resource can be a task or project",
+        required=True,
+    )
+    create_parser.add_argument(
+        "--target",
+        type=str,
+        help="The webhook URL to receive the HTTP POST",
```

Comment:
Can we describe an example of this if they run as per the instructions, just to make it clear that they're specifying a callback address they're setting up with the server file? I believe that'd be something like `http://[computer_address]:3000/receive_webhook` (if we change the route below...)